src/AzureExtension/DevBox/DevBoxInstance.cs

```diff
@@ -53,6 +60,11 @@ public class DevBoxInstance : IComputeSystem
         get; private set;
     }
 
+    public string? DiskSize

```

Comment:
Non-blocking (probably better in a separate PR): Consider grouping IComputeSystem APIs together. I'd wrap them in #region too. I think it would be easier to navigate code that way. "DiskSize" is "public" here (should it be private?), but it's not part of the SDK. There is no way to know that without looking into IComputeSystem definition.

Reply:
I like the suggestion. I have incorporated the changes; are these what you had in mind?